tests/api/api/test_docs.py

```diff
@@ -25,7 +27,7 @@ def test_save_openapi_json(
     path = os.path.abspath(os.getcwd())
     if os.getenv("MLRUN_BC_TESTS_OPENAPI_OUTPUT_PATH"):
         path = os.getenv("MLRUN_BC_TESTS_OPENAPI_OUTPUT_PATH")
-    with open(
-        os.path.join(os.path.join(path, os.getenv("MLRUN_OPENAPI_JSON_NAME"))), "w"
-    ) as file:
+    file_path = os.path.join(path, os.getenv("MLRUN_OPENAPI_JSON_NAME"))
+    with open(file_path, "w") as file:
         file.write(response.text)
+    logger.info(f"Saved openapi file in {file_path}")
```

Comment:
Our logger supports logging kwargs, it's preferred using them over formatted strings:
```suggestion
    logger.info(f"Saved openapi JSON file", file_path=file_path)
```